src/components/ThemeProvider/README.md

```diff
@@ -0,0 +1,230 @@
+---
+name: Theme provider
+category: Structure
+keywords:
+  - theme
+  - provider
+  - color
+  - logo
+  - color scheme
+  - light mode
+  - dark mode
+---
+
+# Theme provider
+
+Theme provider is a component that enables sharing global theme settings throughout the hierarchy of your application. It is included by default as a child of the [app provider component](https://polaris.shopify.com/components/structure/app-provider), but can be used independently to apply a base theme, or to override a parent theme in its children.
+
+---
+
+## Examples
+
+### Theme provider rendered by the app provider
+
+The app provider component renders a theme provider component and a theme. By default, theme falls back to the current design language. The new design language is enabled by passing `{newDesignLanguage: true}` to the `features` prop on the app provider component.
```

Comment:
Wondering if we might want to move away from using "current" and "new" when talking about design language. Could this be confusing and will it need to be changed in the future?

Reply:
This feels like a temporary problem, as this content pretty much goes away when the new design language *is* the design language. My vote is this stays as-is for now, but I am open to any language that can add some clarity.

Reply:
Fair point. What if we keep it but change "current" to "old" to be sure that folks don't mistake "current" for "new"?